src/library/scala/collection/StringOps.scala

```diff
@@ -1,10 +1,12 @@
 package scala
 package collection
```

Comment:
Could be `package scala.collection`, though the statements are not strictly identical any more.

Reply:
please stop.  these comments are not helpful for the PR review.